src/caret_analyze/log_config.yaml

```diff
@@ -0,0 +1,39 @@
+version: 1
+
+formatters:
+  caret_fmt:
+    format: "%(levelname)-8s: %(asctime)s | %(message)s"
+    datefmt: "%Y-%m-%d %H:%M:%S"
+
+handlers:
+  stream_handler:
+    class: logging.StreamHandler
+    level: WARN
+    formatter: caret_fmt
+  file_handler:
+    class: logging.FileHandler
+    level: WARN
+    formatter: caret_fmt
+    filename: ./caret.log
+    mode: w
+    encoding: utf-8
+  null_handler:
+    class: logging.NullHandler
+
+loggers:
+  caret_analyze.architecture.architecture_loaded:
+    level: WARN
+    handlers: [null_handler]
```

Comment:
- Is there any reason you change `handlers` rather than  `level` ?
    - Logger settings for `runtime_loaded` just set `level` as `CRITICAL`. To me, it's simple and natural.
- Probably we should not ignore all logs from `architecture_loaded`
    - For instance, `Duplicated node name` log should be informed to an operator

Why don't we just use `logger.info (or debug)` instead of `logger.warn` for annoying logs ( `Failed to find callback group` and  `Multiple executors using the same callback group were detected.` ) for now ?  So that we can also suppress `Multiple executors using the same callback group were detected` log while we can enable other logs from architecture_loaded which may not have to be missed.
We need to revert log level in the future though.

Reply:
>Is there any reason you change handlers rather than level ?

In this PR, we are considering making minor changes to the configuration file to output logging to a file.
If the logger level is set to `CRITICAL`, no logs will be generated.
Therefore, it will not be possible to output warnings to a file.
For this reason, we think it is desirable to set the logging level to `WARN` instead of `CRITICAL`.
On the other hand, I found some inconsistencies in the current configuration file.
The architecture_loaded and runtime_loaded levels should be the same. 
(fixed in [8d0f0ce](https://github.com/tier4/CARET_analyze/pull/220/commits/8d0f0ce1b31a5a5cac4b473f17b7ec5f2371214b))

> Probably we should not ignore all logs from architecture_loaded

It is true that some of the warnings contain warnings that should not be ignored.
I think it is a good practice to use the filtering feature of logging for the following reasons
- Although there are differences in the importance of warnings within the warn level, none of the warnings should be changed in the info level.
- We want to keep the changes as small as possible.

Reply:
As you mentioned `architecture_loaded` has some warnings which should not be ignored. If we miss such warnings a measurement will fail and we'd like to avoid such a situation especially in field testing.

I understand we should keep the level of all warning messages as `warning` . However, we do want to remove some certain warning messages as importance of these warnings are low.
Could you consider using a custom log level or just using `logger.log(WARNING-1, 'hoge')` may be okay as it's a interim measure.

Reply:
> In this PR, we are considering making minor changes to the configuration file to output logging to a file.

I missed this line.
If the purpose of this PR is just to output logging to a file, do you have a plan to make another PR to suppress warning messages to be ignored ( `Failed to find callback group` and `Multiple executors using the same callback group were detected` )?
If you are adding warning message suppression in this PR, I'd like you to consider the way to keep necessary warning messages from `architecture_loaded` as I mentioned in the previous comment.

Reply:
> In this PR, we are considering making minor changes to the configuration file to output logging to a file.

Sorry, this sentence was inaccurate.
I meant to say "I want to be able to see the suppressed warnings (warnings that can be ignored) if necessary. To provide a way to do this, I intended to make it possible to save warnings generated by architecture_loaded to a file by editing the configuration file.

Just in case I may have misled you by my inaccurate wording, I will share the idea again when using the filter.

---
When using filtering, I was thinking of using the following two handlers.
-  StreamHandler:
    - Output only mandatory warnings.
    - Identification of necessary warnings is implemented by filtering.

- FileHandler:
    - Output the warning contents of the target logger to a file, ignoring unnecessary warnings.

---
If this is not sufficient in the context of field testing, I will implement it by changing the log level of the proposal.

Reply:
Using two handlers makes it easier to check warning messages. However, an operator still needs to investigate a log file (caret.log) where both warning messages to be ignored and warning messages to be checked exist.
It will be difficult for an operator to check it in a car with a very small display. I'm sorry to bother you but could you please change the log level?

Reply:
I understood.
I will implement this by changing the log level.

Reply:
I have changed the logging level of  following warnings to WARN-1.(fixed in [e0a1ee0](https://github.com/tier4/CARET_analyze/pull/220/commits/e0a1ee05656ed775c5510d532651ca77826d0a9a) )
- `Failed to find callback group. callback_group_id:` 
- `Multiple executors using the same callback group were detected. `

It would be helpful if you could check the following two points
- Is the reduction in the number of warnings sufficient?
- Are necessary warnings being suppressed?

If you have any additional warnings that you would like to see suppressed, please comment.

Reply:
Thank you!

>  Is the reduction in the number of warnings sufficient?

Yes.

>  Are necessary warnings being suppressed?

Yes. Also, warnings which should not be ignored are displayed e.g. `Duplicated node name.`